SYCL/Reduction/reduction_span.cpp

```diff
@@ -0,0 +1,113 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//
+// `Group algorithms are not supported on host device.` on Nvidia.
+// XFAIL: hip_nvidia
+
+// TODO: test disabled due to sporadic fails in level_zero:gpu RT.
```

Comment:
Maybe try turning it on initially? I see many fixes and attempts to fix/address those sporadic fails are done, but I did not see tests to be enabled after that.
We can disable the test when/if it starts failing.

Reply:
Ok.  I've reenabled these tests for now; see a7eb1e180bf95dc4c58518cf9709fb0b215bd04d.